hbase-server/src/main/java/org/apache/hadoop/hbase/util/FSUtils.java

```diff
@@ -227,12 +232,8 @@ public static FSDataOutputStream create(Configuration conf, FileSystem fs, Path
           throw new IOException(ite.getCause());
         } catch (NoSuchMethodException e) {
           LOG.debug("DFS Client does not support most favored nodes create; using default create");
-          if (LOG.isTraceEnabled()) LOG.trace("Ignoring; use default create", e);
-        } catch (IllegalArgumentException e) {
-          LOG.debug("Ignoring (most likely Reflection related exception) " + e);
-        } catch (SecurityException e) {
-          LOG.debug("Ignoring (most likely Reflection related exception) " + e);
-        } catch (IllegalAccessException e) {
+          LOG.trace("Ignoring; use default create", e);
+        } catch (IllegalArgumentException | SecurityException |  IllegalAccessException e) {
           LOG.debug("Ignoring (most likely Reflection related exception) " + e);
```

Comment:
nit: while we're here could we pass this exception rather than coercing it to a string?